js/slideshow.js

```diff
@@ -103,6 +103,38 @@
 
 				// check if we moved along while we were loading
 				if (currentImageId === index) {
+                                        // check if not in cache
+                                        if (this.images[index].desc===undefined){
```

Comment:
We need to make sure we only send the request when we have a media type which supports meta data